deployment/src/main/java/io/quarkiverse/loggingmanager/deployment/LoggingManagerProcessor.java

```diff
@@ -55,18 +58,28 @@ void includeRestEndpoints(BuildProducer<RouteBuildItem> routeProducer,
             Handler<RoutingContext> loggerHandler = recorder.loggerHandler();
             Handler<RoutingContext> levelHandler = recorder.levelHandler();
 
-            routeProducer.produce(nonApplicationRootPathBuildItem.routeBuilder()
+            Builder loggerBuilder = nonApplicationRootPathBuildItem.routeBuilder();
+            Builder levelBuilder = nonApplicationRootPathBuildItem.routeBuilder();
+
+            if (managementConfig.enabled) {
```

Comment:
I don't think you need to check the management Config yourself, you can always use management, it will figure out the config and decided if/when to switch to management.

Reply:
What about the part where I prevent the logger and level endpoint from the OpenAPI UI?
I just see that the health end points vanish from OpenAPI UI when management interface is enabled.
I think it is due to the UI not being able to handle the two different ports.